### ISSUE TITLE
reload is broekn

### ISSUE THREAD

**Issue by evilebottnawi** [webpack-contrib/webpack-hot-client#92]
* Operating System: Ubuntu 18.04
* Node Version: 8.11.3
* NPM Version: 6.2.0
* webpack Version: 4.16.1
* webpack-hot-client Version: 4.1.1

### Expected Behavior

Page doesn't reload.

### Actual Behavior

Page should reload.

### Code

https://github.com/webpack-contrib/webpack-serve/blob/master/docs/addons/static-content-options.config.js

### How Do We Reproduce?

1. Run example above.
2. Change something in `app.js` (page has not been reloaded, because `initial: true` -> https://github.com/webpack-contrib/webpack-hot-client/blob/master/lib/client/index.js#L30 and https://github.com/webpack-contrib/webpack-hot-client/blob/master/lib/client/index.js#L78)
3. Change something again (page has been reloaded, all fine)
4. Change again something in `app.js` (page has not been reloaded, because after reload page `initial` was setted in `true` https://github.com/webpack-contrib/webpack-hot-client/blob/master/lib/client/index.js#L30)

Related problem with `css` (`style-loader` and `css-loader`) - first change not applied because `initial: true`.

**Comment by evilebottnawi** [comment]
@shellscape found problem, i have external `WebSocket` (use `ws@5` version) client and communicate with `webpack-hot-client` web socket server, but in `webpack-hot-client` we use `ws@4`, it is break web sockets :confused:  :disappointed: 

What is blocker for update their? We use in own app `ws@5` and can't downgrade, also we need communicate with `webpack-hot-client` web socket server

**Comment by shellscape** [comment]
The server is supposed to be standards compliant, so I don't know why it would break. You'll have to investigate.

**Comment by marcofugaro** [comment]
Yeah I tried running [this example](https://github.com/webpack-contrib/webpack-serve/blob/master/docs/addons/watch-content.config.js) from webpack-serve and it doesn't work anymore
